sway-lib-std/src/hash.sw

```diff
@@ -123,6 +131,12 @@ impl Hash for bool {
     }
 }
 
+impl Hash for Bytes {
+    fn hash(self, ref mut state: Hasher) {
```

Comment:
Misleading name, this does not hash. It adds to the `Bytes`

Reply:
This is the trait